src/@chakra-ui/gatsby-plugin/semanticTokens.ts

```diff
@@ -31,21 +35,41 @@ const semanticTokens = {
 
     // Design System colors
     primary: { _light: "blue.500", _dark: "orange.500" },
+    primaryHighContrast: { _light: "blue.700", _dark: "orange.100" },
+    primaryLowContrast: { _light: "blue.100", _dark: "orange.800" },
     primaryLight: { _light: "blue.100", _dark: "orange.100" },
     primaryDark: { _light: "blue.700", _dark: "orange.800" },
```

Comment:
wondering if we should deprecated these tokens as well @nloureiro

Reply:
@pettinarip I did not have a chance yet to go back in and mark these since Nuno last updated the palette.

Reply:
Yea, not a blocker. We can always mark them later.